docs/index.md

```diff
@@ -1,17 +1,82 @@
-The Smart Managed Pools service is in charge of conducting investment strategies by executing transfer operations in liquidity pools.  This functionality is fed from the output produced by the ML APY Prediction Oracle service, and is in charge of building and submitting the appropriate transactions to execute the orders. At every decision step, a number of things could occur. The service might:
+![SMPKit](images/smart-managed-pools-logo.png){ align=left width="150" }
+The SMPKit helps you build services that execute investment strategies on liquidity pools. This functionality can be fed from the output produced by an APY Prediction service, and it is in charge of building and submitting the appropriate transactions to execute the orders. Services based on the SMPKit can take a number of actions depending on the input strategy. For example, at a given time point, the service could:
```

Comment:
Just go be clear, it does not use the APY.

Reply:
```
transactions to execute the orders
```
Technically they are transactions to rebalance the pool.

Reply:
What is it using to get the strategies?

Reply:
The current version is utilizng the Fear and Greed Index. The strategy is as per [spec](https://drive.google.com/file/d/1_WuEODpEKV7BR3LMKVLxyZUWjcGhHNRS/view).

Reply:
Fixed.